.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,100 @@
+name: NodeJS CDN Code Deploy 
```

Comment:
is there something in this action that is node specific?

Reply:
no actually it's just a typo brought from the old configuration